portfolio/cumulative_gains.go

```diff
@@ -1,47 +1,48 @@
 package portfolio
 
 import (
-	"math"
 	"sort"
 
+	_ "github.com/shopspring/decimal"
```

Comment:
Was this temporary? Can probably delete this line... unless I forgot something. There are a few other places with this too.

Reply:
Yeah during the migration. Removed 👍